relayer/chains/mock/message_handlers.go

```diff
@@ -0,0 +1,90 @@
+package mock
+
+import (
+	"fmt"
+
+	chantypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+	"github.com/cosmos/relayer/v2/relayer/ibc"
+	"github.com/cosmos/relayer/v2/relayer/provider"
+	"github.com/cosmos/relayer/v2/relayer/provider/cosmos"
+	"go.uber.org/zap"
+)
+
+type MsgHandlerParams struct {
+	mcp           *MockChainProcessor
+	PacketInfo    *chantypes.Packet
+	FoundMessages map[ibc.ChannelKey]map[string]map[uint64]provider.RelayerMessage
+}
+
+var messageHandlers = map[string]func(MsgHandlerParams){
+	ibc.MsgTransfer:        handleMsgTransfer,
+	ibc.MsgRecvPacket:      handleMsgRecvPacket,
+	ibc.MsgAcknowledgement: handleMsgAcknowlegement,
+
```

Comment:
We will also need to handle packets coming out of the state machine like in IBC query/Xchain accounts The events come from [`SendPacket`](https://github.com/cosmos/ibc-go/blob/1acb3cc8645422a98babee6c6667485a9cc87366/modules/core/04-channel/keeper/events.go#L130)